java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -474,4 +474,21 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
      * }</pre>
      */
     CompletableFuture<Double[]> zmscore(String key, String[] members);
+
+    /**
+     * Calculates the difference between the first sorted set and all the successive sorted sets at
+     * <code>keys</code> and stores the difference as a sorted set to <code>destination</code>,
+     * overwriting it if it already exists. Non-existent keys are treated as empty sets.
+     *
+     * @see <a href="https://redis.io/commands/zdiffstore/">redis.io</a> for more details.
+     * @param destination Destination key for the resulting sorted set.
+     * @param keys The keys of the sorted sets to compare.
+     * @return The number of members in the resulting sorted set stored at <code>destination</code>.
+     * @example
+     *     <pre>{@code
+     * Long payload = client.zdiffstore("mySortedSet", new String[] {"key1", "key2"}).get();
+     * assert payload > 0; // At least one member differed in "key1" compared to "key2", and this difference was stored in "mySortedSet".
+     * }</pre>
+     */
+    CompletableFuture<Long> zdiffstore(String destination, String[] keys);
```

Comment:
Does it make sense to swap keys and destination?  I think it's easier to understand. Redis uses this ordering because they have a varargs on the keys, but we don't need to restrict ourselves.

Reply:
Don't think so. We have stuck to the Redis ordering up to this point. Does not make sense to change it now.